src/server/object_services/map_deleter.js

```diff
@@ -74,6 +75,19 @@ async function delete_blocks(blocks) {
     }
 }
 
+/**
+ * @param {nb.Chunk[]} chunks
+ */
+async function delete_chunk_parts(chunks) {
+    if (!chunks || !chunks.length) return;
+    const parts = get_all_chunk_parts(chunks).filter(part => !part.to_db().deleted);
+    const part_ids = parts.map(part => part._id);
+    dbg.log0('delete_chunk_parts: parts ', part_ids);
+    await Promise.all([
```

Comment:
No need for promise.all for only one promise,
```suggestion
    await MDStore.instance().delete_parts_by_ids(part_ids)
```